crates/napi/__test__/index.spec.ts

```diff
@@ -394,4 +394,4 @@ test("find node by range", (t) => {
   })
   const byKind = sg.root().find(js.kind("template_string"))
   t.is(node!.id(), byKind!.id())
-})
+})
```

Comment:
I will address it later. This is an inconsistent naming in ast-grep node's interface and rule. Node has the naming `line` instead of `row`. And we should keep consistent naming here. cc @mohebifar

Reply:
@coderabbitai please open a follow-up GitHub issue for this inconsistency.